ee/cis/macos-13/cis-policy-queries.yml

```diff
@@ -238,39 +238,113 @@ spec:
   name: CIS - Ensure DVD or CD Sharing Is Disabled
   platforms: macOS
   platform: darwin
-  description: 
-  resolution: ""
-  # Will likely require MDM/managed_policies. Most of the sharing checks that follow will be essentially the same query but looking at different types of sharing.
-  query: SELECT * from time;
+  description: |
+    DVD or CD Sharing allows users to remotely access the system's optical drive.
+    Disabling DVD or CD Sharing minimizes the risk of an attacker using the optical drive as
+    a vector for attack and exposure of sensitive data.
+  resolution: |
+    Graphical Method:
+    1. Open System Settings
+    2. Select General
+    3. Select Sharing
+    4. Set CD/DVD Sharing to disabled
+  query: |
+    SELECT 1 WHERE NOT EXISTS (
+      SELECT * FROM plist WHERE
+        path = '/var/db/com.apple.xpc.launchd/disabled.plist' AND
+        key = 'com.apple.ODSAgent' AND
+        value = '0'
+    );
```

Comment:
In my machine I had the following settings:
```sh
$ defaults read /var/db/com.apple.xpc.launchd/disabled.plist
{
    "com.apple.CSCSupportd" = 1;
    "com.apple.bootpd" = 1;
    "com.apple.dhcp6d" = 1;
    "com.apple.ftp-proxy" = 0;
    "com.apple.ftpd" = 1;
    "com.apple.mdmclient.daemon.runatboot" = 0;
    "com.apple.nfsd" = 1;
    "com.apple.screensharing" = 1;
    "com.apple.smbd" = 0;
    "com.docker.socket" = 0;
    "com.fleetdm.orbit" = 0;
    "com.google.keystone.daemon" = 0;
    "com.tenablesecurity.nessusagent" = 0;
    "org.apache.httpd" = 1;
}
```
Then I ran:
```sh
$ /usr/bin/sudo /bin/launchctl disable system/com.apple.ODSAgent
```
and now I get:
```sh
$ defaults read /var/db/com.apple.xpc.launchd/disabled.plist
{
    "com.apple.CSCSupportd" = 1;
    "com.apple.ODSAgent" = 1; <<<<<<<<<<<<<<<<<<
    "com.apple.bootpd" = 1;
    "com.apple.dhcp6d" = 1;
    "com.apple.ftp-proxy" = 0;
    "com.apple.ftpd" = 1;
    "com.apple.mdmclient.daemon.runatboot" = 0;
    "com.apple.nfsd" = 1;
    "com.apple.screensharing" = 1;
    "com.apple.smbd" = 0;
    "com.docker.socket" = 0;
    "com.fleetdm.orbit" = 0;
    "com.google.keystone.daemon" = 0;
    "com.tenablesecurity.nessusagent" = 0;
    "org.apache.httpd" = 1;
}
```

So, don't we want to check that an administrator has explicitly disabled the service?
```
SELECT 1 FROM plist WHERE
  path = '/var/db/com.apple.xpc.launchd/disabled.plist' AND
  key = 'com.apple.ODSAgent' AND
  value = '1';
```

Reply:
If the service was never enabled then it won't show up in disabled because there is no launch agent loaded. I can directly check for that but it's not straightforward for all  of the queries.

Reply:
At least that's the behaviour on my M1 Mac and VM, maybe it's different on x86 macOS